full.async/src/full/async.clj

```diff
@@ -78,6 +104,7 @@
         ; doall to check for throwables right away
         (doall)))
 
+;; TODO lazy-seq vs. vector in <<! ?
```

Comment:
I remember I couldn't figure out an easy way to make <<! lazy but probably didn't spend enough time thinking about it.